server/src/test/java/org/kaazing/gateway/server/cluster/StandaloneClusterContextTest.java

```diff
@@ -28,106 +31,157 @@
     @Test
     public void testIMapImplEntryListenerSupportPut() {
         StandaloneClusterContext context = new StandaloneClusterContext();
-        IMap<String, String> imap = context.getCollectionsFactory().getMap("test");
+        IMap<String, String> imap = context.getCollectionsFactory().getMap("test1");
         String value1 = "value1";
         EntryListenerImpl<String, String> listener1 = new EntryListenerImpl<>();
         EntryListenerImpl<String, String> listener2 = new EntryListenerImpl<>();
         imap.addEntryListener(listener1, true);
         imap.addEntryListener(listener2, true);
         imap.put("test1", value1);
-        assertEquals(listener1.addedCount, 1);
-        assertEquals(listener1.udpatedCount, 0);
-        assertEquals(listener1.removedCount, 0);
-        assertEquals(listener2.addedCount, 1);
-        assertEquals(listener2.udpatedCount, 0);
-        assertEquals(listener2.removedCount, 0);
-        assertEquals(imap.get("test1"), value1);
+        try {
+            Thread.sleep(1000);  //pause long enough for listeners to receive events
```

Comment:
Sleeps are bad, especially when we automate the heck out of testing and run on all the different platforms, some of which will have very low CPU.  Is there a way to do this with a latch that is notified when the value gets set, or something similar.  And then set a timeout for the test for the case the latch is never called